README.md

```diff
@@ -68,6 +69,20 @@ After running the command above you will find sedunlocksrv-pba.img in your curre
 - Eject the USB stick and put it aside for now
 - Use the other USB stick for the sedutil rescue system (see next step)
 
+## Optional SSH unlock
+<img width="490" alt="screenshot" src="https://user-images.githubusercontent.com/15635386/235190018-41b16f4f-3d2c-4b3f-a15d-afdcf034a4bd.png">
+
+Optionally SED disks can be unlocked via SSH. To enable this feature follow above build steps with small additions:
+
+- install dropbear (needed to generate dropbear host keys):`apt-get -y install dropbear`
+- create authorized_keys file in `sedunlocksrv-pba/ssh` folder. It should contain public keys of all key pairs allowed to connect to unlocking service. Have a look at provided `authorized_keys.example`
+- run build with SSH option: `./build.sh SSH`
+
+Usage:
+run `ssh -p 2222 tc@IP` --> enter SED password --> reboot.
+
+It uses port `2222` to avoid certificates' conflicts with booted computer and `tc` default Tiny Core Linux user. It only allows to access SED unlocking with any other SSH services disabled.
```

Comment:
What do you mean by: "It only allows to access SED unlocking with any other SSH services disabled."?

Reply:
https://github.com/kapitainsky/sedunlocksrv-pba/blob/3a0513a3608fee895156c3926106f59a9e06db3c/tc/tc-config#L61

- disallow root logins
- disable password logins
- disable password logins for root
- disable local port forwarding
- disable remote port forwarding

Simply off anything what we do not need to do unlocking job.

Reply:
I understand now :)